docker-compose.yml

```diff
@@ -6,28 +6,24 @@ services:
       context: .
       dockerfile: dev.Dockerfile
     env_file: .env.development
-    environment:
-      RUST_LOG: ${RUST_LOG}
     ports:
       - 7878:7878
     volumes:
-      - ${PWD}/src:/app/src
-      - ${PWD}/target:/app/target
-      - ${PWD}/Cargo.lock:/app/Cargo.lock
-      - ${PWD}/Cargo.toml:/app/Cargo.toml
+      - .:/app
+      - hillow_target:/app/target
     restart: always
     depends_on:
       - database
 
   database:
     image: postgres:13
     env_file: .env.development
-    environment:
-      POSTGRES_USER: ${POSTGRES_USER}
-      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-      POSTGRES_DB: ${POSTGRES_DB}
     ports:
       - 5432:5432
     volumes:
-      - ${PWD}/docker/volumes/database:/var/lib/postgresql/data
+      - hillow_database_data:/var/lib/postgresql/data
```

Comment:
I think is better to have a directory for volume related stuff, `hillow_database_data` is not something that Is part of the project itself is related to docker and its files/resources.

I rather having it inside of the `docker/` directory then.

The `docker/` directory is also useful when you define other docker related stuff such as `Dockfile` and initialization scripts like this:

https://github.com/EstebanBorai/okku-server/tree/main/docker

Reply:
Ok, I wiill change this again to use the docker/ directory for those volumes, but then I will also update the .gitignore in order to ignore them.

Reply:
SGTM!